pyproject.toml

```diff
@@ -1,2 +1,51 @@
-[tool.black]
+[tool.ruff]
+indent-width = 4
 line-length = 140
+target-version = "py312"
+
+[tool.ruff.format]
+docstring-code-format = true
+docstring-code-line-length = "dynamic"
+indent-style = "space"
+line-ending = "auto"
+quote-style = "double"
+skip-magic-trailing-comma = true
```

Comment:
I would like to have the option to decide by the trailing comma if I want to split things in seperate lines or not. Setting this to false and split-on-trailing-comma = true does the trick.